prism-theme-template.css

```diff
@@ -0,0 +1,440 @@
+/**
+ * Your theme's name
+ * If this is an adaptation of an existing theme, credit the creator and/or leave a link to it!
+ * Optional: Your name and/or username
+ */
+
+/**
+ * Prism supports IE11, which does not support CSS variables
+ * However, you are encouraged to leave a list of colours you use here
+ * so that when we transition to Prism V2 (and drop support for IE11),
+ * the transition will be a little easier!
+ */
+
+/* The styles used in this template are mostly taken from the One Dark theme */
+
+/* Set the main properties of the code, code blocks, and inline code */
+code[class*="language-"],
+pre[class*="language-"] {
+	background: hsl(220, 13%, 18%);
```

Comment:
About the default colors: I get that they are necessary to get valid CSS but I already know that people are going to forget to change them.

Maybe we could set them all to `unset`?

`unset` is pretty easy to spot in code review and we could even use a linter rule to detect it:

```json
"declaration-property-value-blacklist": {
	"//": ["/\\bunset\\b/"]
}
```

Reply:
Ah this is great! I'll swap them all out.